ext/oci8/oci8_interface.c

```diff
@@ -42,6 +42,29 @@
 #define OCI_STMT_CALL 10
 #endif
 
+/* {{{ proto bool oci_register_taf_callback( resource connection [, string callback] )
```

Comment:
Now it should be:
oci_register_taf_callback( resource connection [, mixed callback]